.github/workflows/reactive-port-against-feature-pack.yml

```diff
@@ -0,0 +1,122 @@
+name: Test this WIP agains the Reactive Feature Pack
```

Comment:
I suppose this file will not be part of the MR in time of merge action, right?

Reply:
Yeah, I'll remove it for the next rebase with your fixes. Probably in the morning